src/components/ContextData/index.jsx

```diff
@@ -0,0 +1,18 @@
+import React,{ createContext, useContext, useState } from 'react'
+import data from '../../data'
+
+export const ContextData = createContext()
+
+export const DataProvider = ({ initalState = {}, children }) => {
+  const [values, setValues] = useState(initalState)
+
+  return (
+    <ContextData.Provider value={{ ...data, values, setValues }}>
+      {children}
+    </ContextData.Provider>
+  );
+}
+
+export const DataConsumer = ContextData.Consumer;
+
+export const useData = () => useContext(ContextData);
```

Comment:
This is a small convenience to prevent you from needing to import both `ContextData` and `useContext` in every component you want to access data in.